deeplearning4j-scaleout/spark/dl4j-spark-ml/src/main/scala/org/deeplearning4j/spark/ml/impl/SparkDl4jNetworkWrapper.scala

```diff
@@ -0,0 +1,117 @@
+package org.deeplearning4j.spark.ml.impl
+
+import java.util
+
+import org.apache.spark.ml.{PredictionModel, Predictor}
+import org.apache.spark.ml.param.ParamMap
+import org.apache.spark.ml.util._
+import org.apache.spark.mllib.linalg.Vector
+import org.apache.spark.mllib.regression.LabeledPoint
+import org.apache.spark.sql.Row
+import org.deeplearning4j.nn.conf.MultiLayerConfiguration
+import org.deeplearning4j.nn.multilayer.MultiLayerNetwork
+import org.deeplearning4j.optimize.api.IterationListener
+import org.deeplearning4j.spark.impl.multilayer.SparkDl4jMultiLayer
+import org.deeplearning4j.spark.ml.utils.{DatasetFacade, ParamSerializer}
+import org.deeplearning4j.util.ModelSerializer
+
```

Comment:
my dang ide took it out, because I have been having a build issue on one of my machines. Thanks for pointing this out.